### PR TITLE
Replace _color with item_color

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Replaces all _color with item_color, for consistency with other codebases.

### CODE REVIEW

File: code/game/machinery/washing_machine.dm [hunk 1]
```diff
@@ -52,15 +52,15 @@
 
 
 	if(crayon)
-		var/_color
+		var/item_color
```
Comment (tigercat2000) [inline code/game/machinery/washing_machine.dm]: uhhhhhhh

File: code/game/machinery/atmoalter/canister.dm [hunk 2]
```diff
@@ -393,7 +393,7 @@ update_flag
 	data["name"] = name
 	data["menu"] = menu ? 1 : 0
 	data["canLabel"] = can_label ? 1 : 0
-	data["_color"] = _color
+	data["item_color"] = item_color
```
Comment (tigercat2000) [inline code/game/machinery/atmoalter/canister.dm]: errrrrrrrrrr

File: code/game/machinery/washing_machine.dm (hunk 1 above)
Comment (Markolie) [inline code/game/machinery/washing_machine.dm]: What seems to be the issue?

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 3]
```diff
@@ -218,18 +218,18 @@ var/global/chicken_count = 0
 	attacktext = "kicks"
 	health = 10
 	var/eggsleft = 0
-	var/_color
+	var/item_color
```
Comment (tigercat2000) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: i don't think chickens are items

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 4]
```diff
@@ -23,7 +23,7 @@
 	density = 0
 	ventcrawler = 2
 	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
-	var/_color //brown, gray and white, leave blank for random
+	var/item_color //brown, gray and white, leave blank for random
```
Comment (tigercat2000) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm]: mice are also probably not items

File: code/game/machinery/washing_machine.dm (hunk 1 above)
Comment (tigercat2000) [inline code/game/machinery/washing_machine.dm]: mass replace replacing things it probably shouldn't is the issue

Comment (Markolie) [inline code/game/machinery/washing_machine.dm]: What does it matter? In this case especially, it's about the color of the crayon, which is an item.

Comment (tigercat2000) [inline code/game/machinery/washing_machine.dm]: eh in this case it's fine, same for the chicken and mice, mostly wanted to point out every unrelated thing changed

the canister is an issue though

**Comment by Fox-McCloud** [comment]
Big thank you!

**Pull request merged**